translate/storage/fpo.py

```diff
@@ -210,7 +210,7 @@ def merge(self, otherpo, overwrite=False, comments=True, authoritative=False):
 
         def mergelists(list1, list2, split=False):
             #decode where necessary
-            if unicode in [type(item) for item in list2] + [type(item) for item in list1]:
+            if str in [type(item) for item in list2] + [type(item) for item in list1]:
```

Comment:
First of all, I didn't write this code and my assumptions might be mistaken.

Reading the code within the `if` my understanding is that if any unicode string was present on any of the lists, then it converts all the other non-unicode strings to unicode. This makes sense to me since when you are merging together two files, one of which is unicode and the other not, you want them to be in the same encoding to merge them, and unicode feels like the right encoding.

@dwaynebailey Perhaps you can shed some light here.